### PR TITLE
Add lang files

### PR DESCRIPTION
**Pull request by jannyHou**
Connect to strongloop-internal/scrum-loopback#1070

### CODE REVIEW

**Comment by jannyHou** [comment]
@slnode test please

**Comment by jannyHou** [comment]
@0candy PTAL. Thanks. Seems CI are all broken :(

**Comment by jannyHou** [comment]
@slnode test please

File: intl/pt_BR/messages.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (0candy) [inline intl/pt_BR/messages.json]: should be `intl/pt/messages.json`

**Comment by 0candy** [comment]
Seems to be missing `tr`, make sure to check that its using the latest chinese files as well from the second round of translations.

**Comment by jannyHou** [comment]
@slnode test please

**Comment by 0candy** [comment]
@slnode test please

**Comment by jannyHou** [comment]
@slnode test please

**Comment by 0candy** [comment]
CI failures being addressed with https://github.com/strongloop/loopback-workspace/pull/311. Merging this PR.

**Pull request merged**